### PR TITLE
Process boolean values

### PR DESCRIPTION
**Pull request by SergeyVartanov**

### CODE REVIEW

File: tests/RestApiSuite/TestPost/content.txt [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
-|FirstName                  |LastName    |
-|Tony                       |Soprano     |
-|Jennifer                   |Melfi       |
+!|post    |${URL}/users    |
+|FirstName|LastName|IsMafia|
+|Tony     |Soprano |True   |
+|Jennifer |Melfi   |False  |
+
+!|script|RestTools                       |
+|check|get_json_str|${URL}/users|[{"FirstName": "Tony", "IsMafia": true, "LastName": "Soprano", "id": 1}, {"FirstName": "Jennifer", "IsMafia": false, "LastName": "Melfi", "id": 2}]|
```
Comment (grigoryvasiliev) [inline tests/RestApiSuite/TestPost/content.txt]: why don't you use http result as table fixture? why do you need get_json_str fixture?

Comment (SergeyVartanov) [reply tests/RestApiSuite/TestPost/content.txt]: http result gives me boolean values in True/False format regardless of how they are stored in the POST data. Now they are stored as "True"/"False" strings that does not work for me.
I need this to be stored as boolean type.

Comment (grigoryvasiliev) [reply tests/RestApiSuite/TestPost/content.txt]: tests are examples of use - so I see here you have introduced a new fixture get_json_str - which is confusing
I suggest to use existing GET fixture plus check RAW result after this get by RAW Result fixture
it also would be better to create separate test for this specific use case

Comment (grigoryvasiliev) [reply tests/RestApiSuite/TestPost/content.txt]: @AntonTimiskov - there is a change in existing behavior of POST fixture - please check whether you expect issues with this in DMP test suite

File: tests/RestApiSuite/TestPost/content.txt [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+!|post    |${URL}/users    |
+|FirstName|LastName|IsMafia|
+|Tony     |Soprano |True   |
+|Jennifer |Melfi   |False  |
+
+
+!|Query:http result as table|${URL}/users    |
+|FirstName                  |LastName|IsMafia|
+|Tony                       |Soprano |True   |
+|Jennifer                   |Melfi   |False  |
```
Comment (mikeplavsky) [inline tests/RestApiSuite/TestPost/content.txt:12]: I revert `http_calls.py` to the original version, and this particular test is green. So why such change?

**Comment by mikeplavsky** [comment]
So the point is True and False must be sent as booleans, not as strings - is this correct?

**Comment by mikeplavsky** [comment]
So I would rename `Test Raw Get` to `Test Booleans In Json` for example

**Comment by mikeplavsky** [comment]
Seems like I closed this by mistake :(

**Pull request closed without merging**